sortinghat/cli/templates/rm.tmpl

```diff
@@ -0,0 +1,5 @@
+{% if is_id_removed %}
+Identity {{ uuid }} removed
+{%- else %}
```

Comment:
The character `-` after the `%` is not present in the other templates, is there a reason for that?

Reply:
This is a special character. When it's not set, a blank line is printed instead of the `else` control tag.